
---
float32 dist_x
float32 dist_y
float32 average_x
float32 average_z
